cypher/cypher-docs/src/docs/dev/glossary.asciidoc

```diff
@@ -296,57 +296,65 @@ The following commands are only executable against the `system` database:
-| <<administration-security-subgraph-read, READ>>                                               | GRAPH                 | Determines whether the properties of an element (node, relationship or both) can be read while executing queries on the specified graph.
-| <<administration-security-administration-dbms-privileges-role-management, REMOVE ROLE>>       | DBMS                  | Determines whether the user can revoke roles.
-| <<administration-security-administration-dbms-privileges-role-management, ROLE MANAGEMENT>>   | DBMS                  | Determines whether the user can create, drop, grant, revoke and show roles.
-| <<administration-security-administration-dbms-privileges-role-management, SHOW ROLE>>         | DBMS                  | Determines whether the user can get information about existing and assigned roles.
-| <<administration-security-administration-database-transaction, SHOW TRANSACTION>>             | Database              | Determines whether a user is allowed to list transactions and queries.
-| <<administration-security-administration-dbms-privileges-user-management, SHOW USER>>         | DBMS                  | Determines whether the user can get information about existing users.
-| <<administration-security-administration-database-startstop, START>>                          | Database              | Determines whether a user can start up a specific database.
-| <<administration-security-administration-database-startstop, STOP>>                           | Database              | Determines whether a user can stop a specific running database.
-| <<administration-security-administration-database-transaction, TERMINATE TRANSACTION>>        | Database              | Determines whether a user is allowed to end running transactions and queries.
-| <<administration-security-administration-database-transaction, TRANSACTION MANAGEMENT>>       | Database              | Determines whether a user is allowed to list and end running transactions and queries.
-| <<administration-security-subgraph-traverse, TRAVERSE>>                                       | GRAPH                 | Determines whether an element (node, relationship or both) can be found and traversed while executing queries on the specified graph.
-| <<administration-security-administration-dbms-privileges-user-management, USER MANAGEMENT>>   | DBMS                  | Determines whether the user can create, drop, modify and show users.
-| <<administration-security-subgraph-write, WRITE>>                                             | GRAPH                 | Determines whether the user can execute write operations on the specified graph.
+|Name                                                                                                     | Category              | Description
```

Comment:
What have you done to change the whole table?

Reply:
It's a bit hard to review :P

Reply:
The new commands where longer so had to add spaces on all lines to get the category column aligned, it is easier to see in Intellij I believe

Reply:
ah, will check it out

Reply:
reviewed it in intellij and had two comments